tsfc/driver.py

```diff
@@ -267,7 +267,7 @@ def name_multiindex(multiindex, name):
     return builder.construct_kernel(kernel_name, impero_c, index_names, quad_rule)
 
 
-def compile_expression_dual_evaluation(expression, to_element, coordinates, *,
+def compile_expression_dual_evaluation(expression, to_element, *,
```

Comment:
`to_element` should eventually become a UFL cofunction whenever they are implemented